src/organisms/neural_network.py

```diff
@@ -11,3 +11,7 @@ def calc_output(self, inputs):
         for layer in self.layers:
             inputs = layer.calc_output(inputs)
```

Comment:
I did just realize this now: why is the return variable called "inputs" when the function is called "calc_**output**"?